raco/language/myrialang.py

```diff
@@ -684,6 +700,9 @@ def num_tuples(self):
     def shortStr(self):
         return "%s" % self.opname()
 
+    def partitioning(self):
```

Comment:
Shouldn't `partitioning` for the Consumer side of an exchange operator reflect the partitioning it imposes on its output (i.e., the actual partitioning of tuples on the worker where it's running)?